internal/state/services.go

```diff
@@ -56,7 +58,11 @@ type EmptyEvidencePool struct{}
 func (EmptyEvidencePool) PendingEvidence(maxBytes int64) (ev []types.Evidence, size int64) {
 	return nil, 0
 }
-func (EmptyEvidencePool) AddEvidence(types.Evidence) error                { return nil }
-func (EmptyEvidencePool) Update(State, types.EvidenceList)                {}
-func (EmptyEvidencePool) CheckEvidence(evList types.EvidenceList) error   { return nil }
+func (EmptyEvidencePool) AddEvidence(context.Context, types.Evidence) error {
```

Comment:
It would be fine to leave these as one-liners too, YMMV.

Reply:
Apparently its not per go standard :) I have to leave it like this.